styles.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+@media (max-width: 600px) {
+  html {
+    font-size: 65%;
+  }
+}
```

Comment:
In my suggestion, your code would look so much better if you add a min-width property to the media query as it will make your webpage more responsive for mobile phone users as well.